### DISCUSSION TITLE
新！

### DISCUSSION THREAD

**Discussion by shundebo** [shundebo/shundebo.github.io#3] [Ideas]
<hr /><em>This discussion was created from the release <a href='https://github.com/shundebo/shundebo.github.io/releases/tag/2023-1-23'>新！</a>.</em>

**Comment by LineXic** [comment]
网站SSL证书没了？

**Reply by shundebo** [reply]
的确，不知道为什么，现在申请

**Reply by LineXic** [reply]
被人打了（不是

**Reply by shundebo** [reply]
> 被人打了（不是

为什么一个星期了还没好（

**Reply by LineXic** [reply]
去申请新的了吗？（指证书

**Reply by shundebo** [reply]
> 去申请新的了吗？（指证书

嗯

**Reply by LineXic** [reply]
能访问了（喜
